pkg/quota/registry.go

```diff
@@ -12,8 +12,9 @@ import (
 // vice versa. If the registry will be used in admission plugin, pass true to forAdmission.
 // See a package documentation of pkg/quota/image for more details.
 func NewRegistry(osClient osclient.Interface, forAdmission bool) kquota.Registry {
+	imageCache := image.NewImageCache()
```

Comment:
@derekwaynecarr in the future I think this is an example of a cache we should have a flyweight for.